.github/workflows/unittest.yml

```diff
@@ -0,0 +1,60 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python
+
+name: unittest
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+      - main
+
+permissions:
+  contents: read
+
+jobs:
+  unittest-single:
+    runs-on: [self-hosted, linux]
```

Comment:
By now, does our self-hosted machine satisfy the requirements from security colleagues?

Reply:
Nope